sirepo/auth_role_moderation.py

```diff
@@ -150,6 +150,8 @@ def _send_request_email(info):
                 )
 
         l = self.absolute_uri(self.uri_for_api("admModerateRedirect"))
+        if not req.req_data.get("reason"):
```

Comment:
This is more robust `len(get("reason", "").strip()) == 0`. When validating user input, make sure it's not a blank.